cmd/cluster.go

```diff
@@ -747,7 +747,7 @@ func getClusterK8sOptions(c *cliclient.MasterClient) []string {
 	setting, _ := c.ManagementClient.Setting.ByID("k8s-version-to-images")
 	var objmap map[string]*json.RawMessage
 
-	json.Unmarshal([]byte(setting.Value), &objmap)
+	_ = json.Unmarshal([]byte(setting.Value), &objmap)
```

Comment:
Looking at this function, it looks like there are a couple of cases where it could fail?

What happens if `ByID` returns an error? If the JSON can't be unmarshaled?

I appreciate that this is apparently unlikely (based on the signature of the function) but if we're not going to return the error, maybe we could return an empty slice if `ByID` returns an error?

Reply:
A couple of lines before actually there was an API call without the err check. Because of this it's probably better to check and return an error (it could be helpful to warn the user about some network misconfiguration).

Done with 05fc2fa.